### PR TITLE
snap: fix versioning

### PR DESCRIPTION
**Pull request by soumyaDghosh**
This is the temporary fix for #733
I will still look into that.

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -1,12 +1,11 @@
 name: saber # you probably want to 'snapcraft register <name>'
 base: core22 # the base snap is the execution environment for this snap
 adopt-info: saber
-version: 0.13.1
```
Comment (adil192) [inline snap/snapcraft.yaml:4]: We should probably leave this in (but with single quotes) anyway

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
 grade: stable # must be 'stable' to release into candidate/stable channels
 confinement: strict # use 'strict' once you have the right plugs and slots
 license: GPL-3.0
 compression: lzo
-icon: assets/icon/icon.png
+icon: snap/assets/icon/icon.png
```
Comment (adil192) [inline snap/snapcraft.yaml:8]: Is this the right path?

File: snap/snapcraft.yaml [hunk 3]
```diff
@@ -36,7 +35,7 @@ parts:
     after: [rustup]
     source: https://github.com/adil192/saber.git
     source-type: git
-    source-tag: v$SNAPCRAFT_PROJECT_VERSION
+    source-tag: v0.13.1
```
Comment (adil192) [inline snap/snapcraft.yaml:38]: And leave this as it was

**Review by adil192 — CHANGES_REQUESTED** [review]

File: snap/snapcraft.yaml [hunk 4]
```diff
@@ -55,6 +54,9 @@ parts:
       - CARGO_HOME: $CRAFT_STAGE/usr/share/rust
       - CARGO_BUILD_JOBS: $CRAFT_PARALLEL_BUILD_COUNT
     override-build: |
+      cd $CRAFT_PART_SRC
+      craftctl set version=$(git describe --tags --abbrev=0 | cut -c 2-)
```
Comment (adil192) [inline snap/snapcraft.yaml:58]: Can we use `$SNAPCRAFT_PROJECT_VERSION` here instead of git?

File: flatpak/com.adilhanney.saber.metainfo.xml [hunk 5]
```diff
@@ -66,7 +66,8 @@
         </screenshot>
     </screenshots>
 
-        <release version="0.13.1" type="development" date="2023-06-18">
+    <releases>
+         <release version="0.13.1" type="development" date="2023-06-18">
```
Comment (adil192) [inline flatpak/com.adilhanney.saber.metainfo.xml:70]: Thanks for spotting this! Just to keep the tree clean, please can you revert this commit and I'll re-commit it to the main branch.

Comment (adil192) [reply flatpak/com.adilhanney.saber.metainfo.xml:70]: Update: recommited in https://github.com/adil192/saber/commit/916db979c5b05221bdcccb340699137af91ed6ac

Comment (soumyaDghosh) [reply flatpak/com.adilhanney.saber.metainfo.xml:70]: Sure, this commit is most probably of no use then. Deleting this PR.

Comment (adil192) [reply flatpak/com.adilhanney.saber.metainfo.xml:70]: Why did you delete the whole PR?

**Pull request closed without merging**